File: sum/sum.c
#include <stdio.h>

int main()
{
    int cn;
    scanf("%d", &cn);
    for (int i = 0; i < cn; i++)
    {
        int a, b, c;
        scanf("%d%d%d", &a, &b, &c);
        if (a + b == c || a + c == b || b + c == a)
        {
            printf("YES\n");
        }

        else
        {
            printf("NO\n");
        }
    }
}

File: sum/sum.py
def main():
    cn = int(input())
    for _ in range(cn):
        a, b, c = map(int, input().split())
        if a + b == c or a + c == b or b + c == a:
            print("YES")
        else:
            print("NO")
    
    
if __name__ == "__main__":
    main()
